tests/Behat/AuthenticationContext.php

```diff
@@ -147,4 +147,30 @@ public function allowExternalUsers(): void {
     $this->configContext->setConfig('oe_authentication.settings', 'assurance_level', 'LOW');
   }
 
+  /**
+   * Configures the CAS server so users are created in login if not exists.
+   *
+   * @Given the site is configured to register users if not exists
```

Comment:
We don't need this step. We should do all operations via UI. This the reason why Behat testing exists.